third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -2198,6 +2310,42 @@ impl Frame {
                             .operand_stack
                             .push(reference.test_variant(info.variant)?)?;
                     },
+                    Bytecode::PackClosure(fh_idx, mask) => {
+                        // TODO(#15664): runtime checks. We should verify that the popped
```

Comment:
BTW I think @ziaptos refactored the runtime checks so that you wouldn't need to add the checks in the core interpretation loop.